x-pack/test/api_integration/apis/management/rollup/rollup.js

```diff
@@ -139,7 +138,11 @@ export default function ({ getService }) {
                   'testCreatedField': {
                     'agg': 'date_histogram',
                     'delay': '1d',
-                    'interval': '24h',
+                    // TODO: Note that we created the job with `interval`, but ES has coerced this to
```

Comment:
Why don't we fix it in this PR?

Reply:
`interval` is deprecated, but still supported. When ES receives `interval` it will coerce it into either `fixed_interval` or `calendar_interval`, which is why we see one of those when getting the rollup capabilities/job from ES. Jen opened https://github.com/elastic/kibana/issues/36306 to update the UI at some point to support creation of a rollup job with either `fixed_interval` or `calendar_interval`, instead of `interval`, but it's not a necessary change at the moment.